test/new-e2e/pkg/utils/e2e/client/agent_client.go

```diff
@@ -30,6 +31,21 @@ func NewHostAgentClient(t *testing.T, host *components.RemoteHost, waitForAgentR
 	return commandRunner, nil
 }
 
+// NewHostAgentClientWithParams creates an Agent client for host install with custom parameters
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Thank you, follow up get rid of the other one and have this as the single source of truth